src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -103,13 +110,19 @@ public function request(string $method, string $url, array $options = []): Respo
             ];
 
             if ('GET' === $method && !$options['body'] && $expectedHeaders === $pushedHeaders) {
+                $this->logger && $this->logger->info(sprintf('Connecting request to pushed response: %s %s', $method, $url));
```

Comment:
1. I would use quote around `%s` (to detect empty method or empty URL for exemple)
2. I would move this to debug log level